cfme/control/explorer/alerts.py

```diff
@@ -391,3 +431,12 @@ class AlertCopy(CFMENavigateStep):
 
     def step(self):
         self.prerequisite_view.configuration.item_select("Copy this Alert", handle_alert=True)
+
+
+@navigator.register(AlertCollection)
```

Comment:
@digitronik do you have any suggestions for a good short name?

Reply:
`Overview` As its just overview of `Alerts`. Its not required comment; but it will good to go with good short name.

Reply:
I agree, however, there is also a page on Monitor->Alerts->All. So we cannot just have `All` for that page since it will conflict with the page on that is already registered to the `alert_collection` called 'All'. Maybe just `MonitorAll` and `MonitorOverview` would be fine